common/views/components/GlobalContextProvider/GlobalContextProvider.tsx

```diff
@@ -87,7 +87,7 @@ const GlobalContextProvider: FunctionComponent<Props> = ({
 };
 
 export function getGlobalContextData(
-  context: GetServerSidePropsContext
+  context: GetServerSidePropsContext | NextPageContext
```

Comment:
When do we pass the `NextPageContext` here?

Reply:
When we use `getInitialProps` for the `_error` page